gradle/dependency-versions.gradle

```diff
@@ -34,7 +34,7 @@
   jodaTimeVersion = "2.2"
   joptSimpleVersion = "3.2"
   junitVersion = "4.12"
-  kafkaVersion = "0.11.0.2"
+  kafkaVersion = "2.0.1"
```

Comment:
After this, through kafka 2.0, we would pull in slf4j-log4j12 transitively. 

Do we know if this would cause logging problems with different log4j-1/log4j-2 combinations that exists in samza.  Or it doesn't matter?

Reply:
We already pull in `slf4j-log4j12` as part of build time dependency in samza-log4j module. Since we have a direct pinned dependency, I suppose it shouldn't matter.

Let me keep an eye for any potential logging issues.